Initium-Odp/war/odp/javascript/script.js

```diff
@@ -2473,13 +2473,14 @@ function switchCharacter(eventObject, characterId, direction)
 	direction = direction || 0;
 	doCommand(eventObject,"SwitchCharacter",{"characterId":characterId, "direction":direction},function()
 	{
+		var charParam = window.characterOverride != null ? ("?char=" + window.characterOverride) : "";
```

Comment:
Fix the null check here. Should be `window.characterOverride!=null && window.characterOverride.length>10`